resources/eventing/charts/nats/templates/20-service.yaml

```diff
@@ -0,0 +1,17 @@
+{{- if .Values.install.enabled }}
+
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ .Values.nats.fullname }}
```

Comment:
(nit) We usually use the helpers functions in the eventing chart to build have a name which consists of the helm chart (`eventing`) and the name of the subchart. What is the intention behind not using it in this chart ? 🤔 

e.g. 
```
{{ include "nats.fullname" . }} # I know this helper does not exist yet
```

That way we would follow the same naming schema

Reply:
done